README.md

```diff
@@ -1 +1,292 @@
+
+```js
+module.exports.sayHelloInEnglish = function() {
+  return 'Hello'
+}
+
+module.exports.sayHelloInTatar = function() {
+  return 'Isänmesez'
+}
+
+module.exports.sayHelloInSpanish = function() {
+  return 'Hola'
+}
+
+module.exports.sayHelloInJapanese = function() {
+  return 'こんにちは' // Kon'nichiwa
+}
+```
+
+The following version is an absolute exact equivalent:
+
+```js
+exports.sayHelloInEnglish = function() {
+  return 'Hello'
+}
+
+exports.sayHelloInTatar = function() {
+  return 'Isänmesez'
+}
+
+exports.sayHelloInSpanish = function() {
+  return 'Hola'
+}
+
+exports.sayHelloInJapanese = function() {
+  return 'こんにちは' // Kon'nichiwa
+}
+```
+
+That's great! In this case `exports` and `module.exports` are the same. What if we want to assign the whole object like we did with the configuration module to the `exports` object?
+
+So this is the working code with `module.exports`. Obviously the code with `module.exports` works, because we used it in the previous section:
+
+```js
+module.exports = function(env) {
+  env = env || process.env.NODE_ENV
+  if (env == 'production') {
+    ...
+  } else if (env == 'testing') {
+    ...
+  } else { // Assume development
+    ...
+  }
+}
+```
+
+What about this code with `exports`? Will it work too?
+
+```js
+exports = function(env) {
+  env = env || process.env.NODE_ENV
+  if (env == 'production') {
+    ...
+  } else if (env == 'testing') {
+    ...
+  } else { // Assume development
+    ...
+  }
+}
+```
+
+Go ahead and try to run it (`environment-exports.js`) with this one-liner command:
+
+```
+node -e "console.log(require('./environment-exports')().apiKey)"
+```
+
+If you're getting `TypeError: require(...) is not a function`, that's right. You are getting an error because now nothing has been exported. By writing `exports = ...`, we **lost** the original value of `exports`!
+
+Note: We lose the reference, because we overwrite the object `exports`. It's okay to overwrite `module.exports` because in this case `module.exports` is a property of the `module` object.
+
+Let's take another example of code to refactor. The code below works, but what happens if we just try to use `exports` instead of `module.exports`?
+
+With `module.exports`:
+
+```js
+module.exports = {
+  sayHelloInEnglish: function() {
+    return 'Hello'
+  },
+  sayHelloInTatar: function() {
+    return 'Isänmesez'
+  },
+  sayHelloInSpanish: function() {
+    return 'Hola'
+  },
+  sayHelloInJapanese: function() {
+    return 'こんにちは' // Kon'nichiwa
+  }
+}
+```
+
+With just `exports`:
+```js
+exports = {
+  sayHelloInEnglish: function() {
+    return 'Hello'
+  },
+  sayHelloInTatar: function() {
+    return 'Isänmesez'
+  },
+  sayHelloInSpanish: function() {
+    return 'Hola'
+  },
+  sayHelloInJapanese: function() {
+    return 'こんにちは' // Kon'nichiwa
+  }
+}
+```
+
+So what's happening with just `exports =...`? It won't work as expected, because we cannot assign exports to something. We lose the reference.
```

Comment:
I'm not sure that this example adds enough new information to warrant keeping -- it feels like it's repeating the above and basically leaving the student more confused.